### ISSUE TITLE
How to configure interiors?

### ISSUE THREAD

**Issue by skitsie** [lfuelling/esx_drugfarms#6]
Okay so when i installed the script, There is no things in the room? In the weed farm there is only a empty room. What should i do? :)

**Comment by skitsie** [comment]
I want the weed plants for example, And in coke room i want the tables with coke on it. Meth lab is a empty room where is the machines? How do i add them? :)

**Comment by lfuelling** [comment]
Sorry for the late reply, for some reason I was not receiving notifications for this repo 🤦‍♂️

*TL;DR*: What you want is configured in bob74_ipl, not this module.

This module only enables blips and teleportation. As stated in the README, you need an IPL loader like [bob74_ipl](https://github.com/Bob74/bob74_ipl) which will load the necessary files from the game.
This is also where you can change what is loaded.

For the weed farm, you need to change the `LoadDefault` function of [bob74_ipl/dlc_bikers/weed.lua](https://github.com/Bob74/bob74_ipl/blob/b6a1f6edcc8c992daa03dc670ef0b557edfb4563/dlc_bikers/weed.lua#L380) to load the stuff you want for each component.

For example to enable the security upgrades, you can change [line 383](https://github.com/Bob74/bob74_ipl/blob/b6a1f6edcc8c992daa03dc670ef0b557edfb4563/dlc_bikers/weed.lua#L383) from
```lua
BikerWeedFarm.Security.Set(BikerWeedFarm.Security.basic)
```
to
```lua
BikerWeedFarm.Security.Set(BikerWeedFarm.Security.upgrade)
```

To see what options are available scroll up a bit, the security suff for example is defined at [line 27](https://github.com/Bob74/bob74_ipl/blob/master/dlc_bikers/weed.lua#L27).

I hope this helps.

**Comment by Brennen11B** [comment]
I dont know if you are still tracking this, but i need assistance. I would like to fill my methlab with everything, im curious if this is correct. 

[EDIT @lfuelling: Moved to #14.]

**Comment by Brennen11B** [comment]
-- Meth lab: 1009.5, -3196.6, -38.99682
exports('GetBikerMethLabObject', function()
    return BikerMethLab
end)

BikerMethLab = {
    interiorId = 247041,

    Ipl = {
        Interior = {
            ipl = "bkr_biker_interior_placement_interior_2_biker_dlc_int_ware01_milo",

            Load = function()
                EnableIpl(BikerMethLab.Ipl.Interior.ipl, true)
            end,
            Remove = function()
                EnableIpl(BikerMethLab.Ipl.Interior.ipl, false)
            end
        }
    },
    Style = {
        none = "",
        empty = "meth_lab_empty",
        basic = {
            "meth_lab_basic",
            "meth_lab_setup"
        },
        upgrade = {
            "meth_lab_upgrade",
            "meth_lab_setup"
        },

        Set = function(style, refresh)
            BikerMethLab.Style.Clear(false)

            if style ~= "" then
                SetIplPropState(BikerMethLab.interiorId, style, true, refresh)
            else
                if refresh then
                    RefreshInterior(BikerMethLab.interiorId)
                end
            end
        end,
        Clear = function(refresh)
            SetIplPropState(BikerMethLab.interiorId, {
                BikerMethLab.Style.empty,
                BikerMethLab.Style.basic,
                BikerMethLab.Style.upgrade
            }, false, refresh)
        end
    },
    Security = {
        none = "",
        upgrade = "meth_lab_security_high",

        Set = function(security, refresh)
            BikerMethLab.Security.Clear(false)

            if security ~= "" then
                SetIplPropState(BikerMethLab.interiorId, security, true, refresh)
            else
                if refresh then
                    RefreshInterior(BikerMethLab.interiorId)
                end
            end
        end,
        Clear = function(refresh)
            SetIplPropState(BikerMethLab.interiorId, BikerMethLab.Security.upgrade, false, refresh)
        end
    },
    Details = {
        production = "meth_lab_production", -- Products

        Enable = function(details, state, refresh)
            SetIplPropState(BikerMethLab.interiorId, details, state, refresh)
        end
    },

    LoadDefault = function()
        BikerMethLab.Ipl.Interior.Load()
        BikerMethLab.Style.Set(BikerMethLab.Style.upgrade)
        BikerMethLab.Security.Set(BikerMethLab.Security.none)
        BikerMethLab.Details.Enable(BikerMethLab.Details.production, false)

        RefreshInterior(BikerMethLab.interiorId)
    end
}
